Python_Engine/Compute/RemoveVirtualenv.cs

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+
+namespace BH.Engine.Python
+{
+    public static partial class Compute
+    {
+        [Description("Remove a virtualenv of the given name.")]
+        [Input("name", "The name of this virtualenv to remove.")]
+        [Input("run", "Run the installation process for this virtualenv.")]
+        [Output("success", "True if the environment of the given name has been removed.")]
+        public static bool RemoveVirtualenv(
+            string name,
+            bool run = false
+        )
+        {
+            if (run)
+            {
```

Comment:
Please remove indented block. Perhaps consider adding a warning to alert user to toggle `run` to `true`.
```suggestion
            if (!run)
            {
               BH.Engine.Base.Compute.RecordWarning($"Please toggle `{nameof(run)}` to true.");
               return null;
            }
```